### PR TITLE
Update documentation.md

### PR DESCRIPTION
**Pull request by felipericieri**
Hey folks 👋
Great work on the Elixir School!
I’ve found some typos in the docs, as follows.

...
Corrects typos, grammar and suggests format

### CODE REVIEW

File: en/lessons/basics/documentation.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ We specify the `only: :dev` key-value pair as we don't want to download and comp
 
 Earmark is a Markdown parser for the Elixir programming language that ExDoc utilizes to turn our documentation within `@moduledoc` and `@doc` to beautiful looking HTML.
 
-It is worth noting at this point that you change the markup tool to Cmark if you wish, but you will need to do a little more configuration which you can read about [here](https://hexdocs.pm/ex_doc/ExDoc.Markdown.html#module-using-cmark).
+It is worth nothing at this point that you change the markup tool to Cmark if you wish, but you will need to do a little more configuration which you can read about [here](https://hexdocs.pm/ex_doc/ExDoc.Markdown.html#module-using-cmark).
```
Comment (tajchumber) [inline en/lessons/basics/documentation.md:212]: `noting` is the correct word here

Comment (doomspork) [inline en/lessons/basics/documentation.md:212]:
```suggestion
It is worth nothing at this point that you change the markup tool to Cmark if you wish, but you will need to do a little more configuration which you can read about [here](https://hexdocs.pm/ex_doc/ExDoc.Markdown.html#module-using-cmark).
```

Comment (doomspork) [reply en/lessons/basics/documentation.md:212]: As @tajchumber mentioned, `noting` is the correct word here. We are taking note.

Comment (doomspork) [inline en/lessons/basics/documentation.md:212]:
```suggestion
It is worth noting at this point that you change the markup tool to Cmark if you wish, but you will need to do a little more configuration which you can read about [here](https://hexdocs.pm/ex_doc/ExDoc.Markdown.html#module-using-cmark).
```

**Review by doomspork — CHANGES_REQUESTED** [review]

**Review by doomspork — APPROVED** [review]

**Pull request merged**